From 892587db96f77b057c43c352b9648a0c8771eef0 Mon Sep 17 00:00:00 2001
From: Francois Choleau <francois.choleau@st.com>
Date: Wed, 19 Jun 2024 11:02:33 -0400
Subject: [PATCH 616/841] feat(stm32mp23-fdts): update STM32MP235F-DK board

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git e3e2132c31c4bbb396f52af12a04397063dadde9 feat(stm32mp23-fdts): update STM32MP235F-DK board]
Signed-off-by: Francois Choleau <francois.choleau@st.com>
Change-Id: I2e0d54f3b52acbb25a2a0b20da4ee90e355e2473
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/388612
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Yann GAUTIER <yann.gautier@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 fdts/stm32mp235f-dk-ca35tdcid-rcc.dtsi | 1 -
 fdts/stm32mp235f-dk.dts                | 7 +++++++
 2 files changed, 7 insertions(+), 1 deletion(-)

diff --git a/fdts/stm32mp235f-dk-ca35tdcid-rcc.dtsi b/fdts/stm32mp235f-dk-ca35tdcid-rcc.dtsi
index 79f405c50..248a7ae2d 100644
--- a/fdts/stm32mp235f-dk-ca35tdcid-rcc.dtsi
+++ b/fdts/stm32mp235f-dk-ca35tdcid-rcc.dtsi
@@ -95,4 +95,3 @@
 		};
 	};
 };
-
diff --git a/fdts/stm32mp235f-dk.dts b/fdts/stm32mp235f-dk.dts
index 76c0d3915..775de4e9e 100644
--- a/fdts/stm32mp235f-dk.dts
+++ b/fdts/stm32mp235f-dk.dts
@@ -47,6 +47,12 @@
 	};
 };
 
+&ddr {
+	vdd1-supply = <&vdd1_ddr>;
+	vdd2-supply = <&vdd2_ddr>;
+	vddq-supply = <&vdd2_ddr>;
+};
+
 &hash {
 	status = "okay";
 };
@@ -126,6 +132,7 @@
 				regulator-name = "vdd1_ddr";
 				regulator-min-microvolt = <1800000>;
 				regulator-max-microvolt = <1800000>;
+				regulator-enable-ramp-delay = <1000>;
 			};
 			vdd3v3_usb: ldo4 {
 				regulator-name = "vdd3v3_usb";
-- 
2.47.2

